lib/commands.py

```diff
@@ -540,23 +541,42 @@ def _mktx(self, outputs, fee=None, change_addr=None, domain=None, nocheck=False,
         if not unsigned:
             run_hook('sign_tx', self.wallet, tx)
             self.wallet.sign_transaction(tx, password)
+            if addtransaction:
```

Comment:
Why does this commit contain cherry-picks from master.  Please learn to use the `git rebase` command.  This needs a rebase.

Reply:
Ok.